spec/models/user_spec.rb

```diff
@@ -74,11 +74,31 @@ def user_error_message(*keys)
         )
 
         expect(SidebarSectionLink.where(linkable_type: 'Tag', user_id: user.id).pluck(:linkable_id)).to contain_exactly(
-          tag.id,
-          hidden_tag.id
+          tag.id
         )
       end
 
+      it "Should only receive new categories they didn't previously have access to" do
```

Comment:
```suggestion
      it "should create and remove the right sidebar section link records when user is promoted/demoted as an admin" do
```

I think this provides a more complete context about the test because the current description does not mention anything about admin promotion/demotion.